packages/components/.storybook/preview-head.html

```diff
@@ -140,4 +140,8 @@
   h2#stories + [id^='anchor--components-'].sb-anchor {
```

Comment:
You can add the selectors after the first one so:
`h2#stories + [id^='anchor--components-'].sb-anchor, h2#stories + [id^='anchor--styles-'].sb-anchor`